ada_2_fhir-r4/zibs2020/payload/0.0/nl-core-TobaccoUse.xsl

```diff
@@ -0,0 +1,182 @@
+                                    </xsl:call-template>
+                                </xsl:attribute>
+                            </end>
+                        </xsl:if>
+                    </effectivePeriod>
+                </xsl:if>
+                
+                <xsl:for-each select="tabak_gebruik_status">
+                    <valueCodeableConcept>
+                        <xsl:call-template name="code-to-CodeableConcept"/>
+                    </valueCodeableConcept>
+                </xsl:for-each>
+                
+                <xsl:for-each select="toelichting">
+                    <note>
+                        <text>
+                            <xsl:attribute name="value" select="./@value"/>
+                        </text>
+                    </note>
+                </xsl:for-each>
+                
+                <xsl:for-each select="soort_tabak_gebruik">
+                    <component>
+                        <code>
+                            <coding>
+                                <system value="http://snomed.info/sct"/>
+                                <code value="53661000146106"/>
+                                <display value="Type of tobacco used"/>
+                            </coding>
+                        </code>
+                        <valueCodeableConcept>
+                            <xsl:call-template name="code-to-CodeableConcept"/>
+                        </valueCodeableConcept>
+                    </component>
+                </xsl:for-each>
+                
+                <xsl:for-each select="waarneming_gebruik/hoeveelheid">
+                    <component>
+                        <code>
+                            <coding>
+                                <system value="http://snomed.info/sct"/>
+                                <code value="266918002"/>
+                                <display value="Tobacco smoking consumption"/>
+                            </coding>
+                        </code>
+                        <valueQuantity>
+                            <xsl:call-template name="hoeveelheid-to-Quantity">
+                                <xsl:with-param name="in" select="."/>
+                            </xsl:call-template>
+                        </valueQuantity	>
+                    </component>
+                </xsl:for-each>
+                
+                <xsl:for-each select="waarneming_gebruik/pack_years">
+                    <component>
+                        <code>
+                            <coding>
+                                <system value="http://snomed.info/sct"/>
+                                <code value="401201003"/>
+                                <display value="pakjaren"/>
+                            </coding>
+                        </code>
+                        <valueInteger>
+                            <xsl:attribute name="value" select="./@value"/>
+                        </valueInteger>
```

Comment:
Blijkbaar wisselen we van Quantity naar Integer, toelichting mist een beetje in release notes

Reply:
Toegevoegd in de release-notes